src/transformers/onnx/features.py

```diff
@@ -286,6 +287,7 @@ class FeaturesManager:
         ),
         "vit": supported_features_mapping("default", "image-classification", onnx_config_cls=ViTOnnxConfig),
         "beit": supported_features_mapping("default", "image-classification", onnx_config_cls=BeitOnnxConfig),
+        "deit": supported_features_mapping("default", "image-classification", onnx_config_cls=DeiTOnnxConfig),
```

Comment:
I saw that DeiT also have a `DeiTForMaskedImageModeling` implementation, is it a feature also available in onnx ?

Reply:
BeiT also has BeiTForMaskedImageModeling, but there isn't anything for that in features.py

Reply:
You can add something like 

```
"masked-im": AutoModelForMaskedImageModeling
```

to the `_TASKS_TO_AUTOMODELS` attribute of the `FeaturesManager`. I don't think we currently support this feature for TensorFlow, so no need to include this in `__TASKS_TO_TF_AUTOMODELS`.

Feel free to then include this feature as part of `vit`, `deit`, and `beit`